File: techpro2710/2Darrays.js
//3x3 table

const table = [
[1, 2, 3],
[4, 5, 6],
[7, 8, 9]
];

//Access elements
console.log(table[0][0]); //1
console.log(table[1][2]); //6

//Iterate
for (let row of table) {
    for (let value of row){
        console.log(value + " ");
    }
}

File: techpro2710/2Darrays.py
table = [
[1, 2, 3, 4],
[5, 6, 7, 8],
[9, 10, 11, 12]
]

# Pick an element
# print(table[0][0]) # 1
# Pick an entire row
# print(table[0]) # [1, 2, 3, 4]

for row in table:
    for value in row:
        print(value, end=" ")
    print()

